tests/test-sessions/test-sessions-common/pom.xml

```diff
@@ -29,10 +29,13 @@
       <artifactId>jetty-client</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+    </dependency>
     <dependency>
       <groupId>org.eclipse.jetty</groupId>
       <artifactId>jetty-slf4j-impl</artifactId>
```

Comment:
Must have `scope=test`

Reply:
I cleaned up all of the `/tests/test-sessions/test-*-sessions/` projects.
Improved the logging on infinispan and hazelcast testing in the process too.